consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/validation/ProposalPayloadValidator.java

```diff
@@ -89,4 +104,58 @@ private boolean validateBlock(final Block block) {
 
     return true;
   }
+
+  /*
+    Validate CMS in block header (only for PKI-mode)
+  */
+  private boolean validateCms(final Block block) {
```

Comment:
Think this would be nicer if it was separated out as it's own validator probably created in the ProposalValidator just like the ProposalPayloadValidator is